pythnet-price-feeds/cosmwasm.md

```diff
@@ -18,25 +39,36 @@ The mechanism by which price feeds are updated on Cosmwasm is explained [here](.
 
 Pyth is currently available on the following cosmwasm chains:
 
-### Mainnet
+### Mainnet (Stable)
```

Comment:
Suggest organizing this into two sections (stable & edge), then in each table including (testnet) or (mainnet) by the network name, like this:

```
### Stable

| ...
Injective Mainnet | ... |
| Osmosis Mainnet | ... |
| --- | --- |  <- separator between mainnets and testnets
| Injective Testnet | ... |
...
```

The rationale for this is that the distinction that Pyth makes is between stable and edge, so testnet / mainnet shouldn't be emphasize in the reader's mind.